.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "rules": {
+    },
+    "ignorePatterns": ["**/*.test.js","**/*.cy.js","cypress.config.js"],
+    "overrides": [
```

Comment:
You should have two overrides, one for jest and one for cypress. This approach may provide mixed results.